src/app/static/src/app/store/baseline/actions.ts

```diff
@@ -3,13 +3,7 @@ import {BaselineState} from "./baseline";
 import {RootState} from "../../root";
 import {api} from "../../apiService";
 import {PjnzResponse, ValidateBaselineResponse} from "../../generated";
-
-export type BaselineActionTypes = "PJNZUpdated" | "ShapeUpdated" | "PopulationUpdated" | "Validated"
```

Comment:
Because we now have the enum values (which correspond exactly to these). It's true could pass "keyof BaselineMutation" as a type to the api service for type checking there, could add that.